test_fixtures/valid/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:pedantic/analysis_options.1.8.0.yaml
```

Comment:
Maybe I'm missing something, but it looks like this file might not be used in any of the test fixtures?

Reply:
Oh, never mind, I see what's going on here.